### DISCUSSION TITLE
Opening vscode will generate__ Unconfig_ What's wrong with vite.config.ts

### DISCUSSION THREAD

**Discussion by SKT1yang** [unocss/unocss#2716] [Q&A]
My vite configuration is in the form of importing an npm package, which includes unocss config configuration. Every time I open vscode, it will generate__ Unconfig_ Vite.config.ts, what is the operation of adding files directly to user projects

**Comment by serkodev** [comment]
Is it the similar issue with https://github.com/unocss/unocss/issues/1432 ?

If possible, could you please share your source code? This would make it much easier for everyone to understand the details of your issue.
